app/assets/javascripts/velumMinionPoller.js

```diff
@@ -0,0 +1,123 @@
+// the semi-colon before function invocation is a safety net against concatenated
+// scripts and/or other plugins which may not be closed properly.
+;( function( $, window, document, undefined ) {
+
+    "use strict";
+
+        // undefined is used here as the undefined global variable in ECMAScript 3 is
```

Comment:
Sure, then put a comment on top stating that the skeleton of this poller comes from that URL 😉